docs/UserWallet.rst

```diff
@@ -0,0 +1,159 @@
+.. _UserWallet:
+
+#####################
+Managing your wallet
+#####################
+
+=======
+Wallet
+=======
+
+
+General information
+~~~~~~~~~~~~~~~~~~~~
+
+Cryto-currency
+A cryptocurrency (or crypto currency) is a digital asset designed to work as a medium of exchange that uses strong cryptography to secure financial transactions, control the creation of additional units, and verify the transfer of assets.
+
+A cryptocurrency wallet is a software program that uses digital encryption keys to interact with a blockchain network to enable users to send and receive digital currency and monitor their balance. 
+For example, using Bitcoin or any other cryptocurrency, you need to have a digital wallet.
+
+
+
+How to access to Wallet interface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+You can access your personal eXo wallet in one of the three following ways :
+
+
+1- From your display name on the top navigation bar then "My wallet" from the drop-down menu.
+
+|image0|
+
+2- From your profile then "My wallet" application.
+
+|image1|
+
+
+3- From "My application" of the left navigation then "My wallet".
+
+|image2|
+
+
+How to create your wallet
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first thing to do is to create your wallet by setting a password. The password must contain at least 8 characters. 
+
+|image3|
+
+Once your password is set, your wallet is created but you will not be allowed to make transactions until an administrator approves it. The wallet's interface is nevertheless available and you may request funds from other users.
+
+|image1|
+
+The following message is displayed : " Almost done! Your wallet will be ready to use once an administrator approves it.".
+
+A warning message appears behind the label "My wallet" 
+
+|image5|
+
+
+Wallet security
+~~~~~~~~~~~~~~~~
+
+Each wallet uses a pair of public and private keys. The public key is used ad the address on the blockchain network. The private key must be secured as it allows to do any operation in the blockchain account (such as stilling your funds!). Without the private key, access to the funds is impossible. Thus, it's important to secure the private key and save in case of loss  :
+
+- Click on security icon to display security popup
+
+|image6|
+
+- From security popup, click on "Manage keys". A new tab "Manage keys" is displayed in the popup. Click on "Back up your wallet".
+
+|image7|
+
+- A new popup "Backup wallet" is available with the following warnig message : *"Please make sure to keep this private key in a safe place where nobody else can get it (Write it in a piece of paper and hide it for example). 
```

Comment:
I'm not sure reiterating the warning verbatim here is useful (it may even be changed from crowdin). It's probably enough to tell that a warning is displayed to remind you the importance of the private key.